### ISSUE TITLE
Can plugin be used on DLA？

### ISSUE THREAD

**Issue by YuKDseele** [NVIDIA/trt-samples-for-hackathon-cn#107]
I want my model to run independently on DLA, but I found that some layers aren't supported. Is it possible to create a plugin to enable DLA to run those layers?

**Comment by wili-65535** [comment]
No, plugin layer is not supported by DLA (https://docs.nvidia.com/deeplearning/tensorrt/developer-guide/index.html#dla_layers).
